### PR TITLE
Privacy policy and cookies

### PR DESCRIPTION
**Pull request by cheedinmar**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,21 +1,32 @@
 <template>
   <div class="flex flex-col min-h-screen">
+ <div class=" " :class="acceptCookie ? '' : 'opacity-70 pointer-events-none' ">
```
Comment (teebarg) [inline src/App.vue:3]: What does this do? Let's add a comment

File: src/components/CookieAlert.vue [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      Accept
+    </button>
+  </div>
+</template>
+<script>
+export default {
+  methods: {
+    AcceptCookie() {
+    window.localStorage.setItem('acceptCookie', true)
+     window.location.reload()
```
Comment (teebarg) [inline src/components/CookieAlert.vue:20]: We don't want to reload after accepting, it should just remove the banner

File: src/views/SignUp.vue [hunk 3]
```diff
@@ -186,80 +205,82 @@ export default {
   },
   data() {
     return {
+      validRegex:
+        /^(\+234|234|0)(701|702|703|704|705|706|707|708|709|802|803|804|805|806|807|808|809|810|811|812|813|814|815|816|817|818|819|909|908|901|902|903|904|905|906|907)([0-9]{7})$/,
```
Comment (teebarg) [inline src/views/SignUp.vue:209]: Interesting regex here, what does it do?

Comment (cheedinmar) [reply src/views/SignUp.vue:209]: phone number validation for Nigeria lines

**Review by teebarg — APPROVED** [review]

**Pull request merged**